exchanges/bitfinex/bitfinex_test.go

```diff
@@ -1046,56 +1046,131 @@ func TestGetDepositAddress(t *testing.T) {
 	}
 }
 
-func setupWs() {
-	var dialer websocket.Dialer
-	err := b.Websocket.AuthConn.Dial(&dialer, http.Header{})
-	if err != nil {
-		log.Fatal(err)
-	}
-	go b.wsReadData(b.Websocket.AuthConn)
-	go b.WsDataHandler()
-}
+// ----------------------------------------------------------------------------------------------------------------------------
+// End RealTests
 
+// TestWs tests have the ability to impact your orders on the exchange. Enable canManipulateRealOrders to run them
+// ----------------------------------------------------------------------------------------------------------------------------
 // TestWsAuth dials websocket, sends login request.
 func TestWsAuth(t *testing.T) {
-	if !b.Websocket.IsEnabled() && !b.API.AuthenticatedWebsocketSupport {
-		sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+	if !b.Websocket.IsEnabled() {
+		t.Skip(stream.WebsocketNotEnabled)
+	}
+	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
+	if !b.API.AuthenticatedWebsocketSupport {
+		t.Skip("Authentecated API support not enabled")
+	}
+	setupWs(t)
+	assert.True(t, b.Websocket.CanUseAuthenticatedEndpoints(), "CanUseAuthenticatedEndpoints should be turned on")
+
+	// An upcoming PR may change this to a specific datatype
```

Comment:
Where's your sense of suspense? Of mystery? Of untapped potential? 🧙 
Okay, yeah.

The reason was so that if this starts failing it'll be clear what to do.
However that's down to the new PR to get right anyway and should get caught automatically by it.

Fixed 5a15a564